invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/PatronDetails/components/ItemsSearch/ItemsSearch.js

```diff
@@ -175,7 +175,7 @@ ItemsSearch.propTypes = {
   fetchUpdatedCurrentLoans: PropTypes.func.isRequired,
   clearResults: PropTypes.func.isRequired,
   checkoutItem: PropTypes.func.isRequired,
-  patron: PropTypes.number.isRequired,
+  patron: PropTypes.string.isRequired,
```

Comment:
I think the patron id is an integer in the database so maybe it makes sense to keep it like that here too? Also, I think it would make a lot more sense if this prop was named `patron_id` or similar.